user_signup/signup.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Sign Up</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<h2>Signup</h2>
+	<form method='post'>
+		<table>
```

Comment:
Why'd you use a table here?  Usually, you'll only want to use the `<table>` tag when you're actually laying out data in a table, rather than to specify a layout.